src/csharp/Microsoft.Spark.E2ETest/Microsoft.Spark.E2ETest.csproj

```
@@ -10,6 +10,10 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.Spark.Extensions.Delta.UnitTest" />
```

```suggestion
    <InternalsVisibleTo Include="Microsoft.Spark.Extensions.Delta.E2ETest" />
```
Since these are end to end tests, we should rename this project.